dbms/src/Storages/Transaction/KVStore.cpp

```diff
@@ -232,52 +233,11 @@ RegionManager::RegionWriteLock KVStore::genRegionMgrWriteLock(const KVStoreTaskL
 }
 
 EngineStoreApplyRes KVStore::handleWriteRaftCmd(
-    raft_cmdpb::RaftCmdRequest && request,
```

Comment:
The debugging method with `raft_cmdpb::RaftCmdRequest` is moved to `dbgTools.cpp`